features/support/webmock.rb

```diff
@@ -5,7 +5,8 @@
 
 require_relative 'capybara_driver_helper'
 
-selenium_url = URI.parse ENV.fetch('SELENIUM_URL', 'http://localhost:4444/wd/hub')
-app_host_url = URI.parse Capybara.app_host
+URI.parse ENV.fetch('SELENIUM_URL', 'http://localhost:4444/wd/hub')
```

Comment:
Rubocop was failing and it was autocorrected by rubocop

Reply:
I bet rubocop was failing because you had values that you are not using. So the autofix just removed the assignment. Please don't use autofix on rubocop if you don't review the changes.